### PR TITLE
[plugin.video.aswim] 3.0.4

### PR DESCRIPTION
**Pull request by tylerszabo**
### Description

- Update for website change
- Add setting for hiding clips
Please seek for approval from original author @learningit before merging

### Checklist:

- [x] My code follows the [add-on rules](http://kodi.wiki/view/Add-on_rules) and [piracy stance](http://kodi.wiki/view/Official:Forum_rules#Piracy_Policy) of this project. 
- [x] I have read the [CONTRIBUTING](https://github.com/xbmc/repo-plugins/blob/master/CONTRIBUTING.md) document
- [x] Each add-on submission should be a single commit with using the following style: [plugin.video.foo] v1.0.0

Additional information :
- Submitting your add-on to this specific branch makes it available to any Kodi version equal or higher than the branch name with the applicable Kodi dependencies limits.
- [add-on development](http://kodi.wiki/view/Add-on_development) wiki page.
- Kodi [pydocs](http://kodi.wiki/view/PyDocs) provide information about the Python API
- [PEP8](https://www.python.org/dev/peps/pep-0008/) codingstyle which is considered best practice but not mandatory.
- This add-on repository has automated code guideline check which could help you improve your coding. You can find the results of these check at [Codacy](https://www.codacy.com/app/Kodi/repo-plugins/dashboard). You can create your own account as well to continuously monitor your python coding before submitting to repo.
- Development questions can be asked in the [add-on development](http://forum.kodi.tv/forumdisplay.php?fid=26) section on the Kodi forum.

### CODE REVIEW

File: plugin.video.aswim/addon.xml [hunk 1]
```diff
@@ -21,7 +21,7 @@
     <license>GNU GENERAL PUBLIC LICENSE. Version 2, June 1991</license>
     <website>www.adultswim.com</website>
     <source>https://github.com/learningit/plugin.video.aswim</source>
```
Comment (Rechi) [inline plugin.video.aswim/addon.xml]: This repo doesn't exist.

**Comment by Lunatixz** [comment]
@learningit

File: plugin.video.aswim/addon.xml (hunk 1 above)
Comment (tylerszabo) [reply plugin.video.aswim/addon.xml]: @learningit how would you like me to proceed? I think a good solution would be to alter the string to point to this repo but I don't want to change this if you're still planning to maintain the current location.

**Comment by learningit** [comment]
I already have an update for this. I'm just not in a place where I can reliably do a PR until next week. I don't want to add any settings to add-ons at this point.

**Comment by learningit** [comment]
At any rate this us the correct repo:
https://github.com/learningit/repo-plugins/tree/krypton/plugin.video.aswim

File: plugin.video.aswim/addon.xml (hunk 1 above)
Comment (learningit) [reply plugin.video.aswim/addon.xml]: Please see comments below

**Comment by tylerszabo** [comment]
NP, thanks for taking a look.

**Comment by learningit** [comment]
Tyler, please submit your PR to the repo I give above. I'll see what differences there are with my updated version and incorporate what I can.

**Comment by tylerszabo** [comment]
Done, I fast-forwarded from the upstream branch, first, to avoid merge conflicts (https://github.com/learningit/repo-plugins/tree/krypton/plugin.video.aswim was at 3.0.2). If the branch is fast-forwarded first that diff will look a lot cleaner and only have the 1 change. Unfortunately I don't know a way to do a PR for a fast forward that won't leave a PR commit for the merge on top.

**Pull request closed without merging**